2022-04-27 10:30:57,816 INFO  ==============================================================
2022-04-27 10:30:57,817 INFO  Id:    tedge_connect_test_positive
2022-04-27 10:30:57,819 INFO  Dir:   tedge_connect/
2022-04-27 10:30:57,819 INFO  Title: Validate tedge connect c8y --test (positive case)
2022-04-27 10:30:57,820 INFO  ==============================================================
2022-04-27 10:30:57,826 INFO  EnvironmentC8y Setup
2022-04-27 10:30:58,440 INFO  Executed systemctl<serv_mapper1>, exit status 3
2022-04-27 10:30:59,063 WARN  Executed sudo<tedge_connect_c8y>, exit status 1
2022-04-27 10:30:59,067 INFO  Contents of tedge_connect_c8y.err: 
2022-04-27 10:30:59,069 INFO    Error: failed to connect Cumulocity cloud.
2022-04-27 10:30:59,069 INFO    Caused by:
2022-04-27 10:30:59,070 INFO        A value for `c8y.url` is missing.
2022-04-27 10:30:59,071 INFO            A value can be set with `tedge config set c8y.url <value>`
2022-04-27 10:30:59,071 INFO    -----
2022-04-27 10:30:59,072 INFO  
2022-04-27 10:30:59,112 WARN  sudo<tedge_connect_c8y> returned non-zero exit code 1 ... blocked 
2022-04-27 10:30:59,113 WARN  Aborted test due to BLOCKED outcome
2022-04-27 10:30:59,118 INFO  
2022-04-27 10:30:59,119 INFO  cleanup:
2022-04-27 10:30:59,736 INFO  Executed sudo<tedge_disconnect_c8y>, exit status 0
2022-04-27 10:31:00,308 INFO  Executed systemctl<serv_mapper5>, exit status 3
2022-04-27 10:31:00,312 INFO  
2022-04-27 10:31:00,313 INFO  Test duration: 2.50 secs
2022-04-27 10:31:00,314 INFO  Test final outcome:  BLOCKED
2022-04-27 10:31:00,314 INFO  Test outcome reason: sudo<tedge_connect_c8y> returned non-zero exit code 1
2022-04-27 10:31:00,315 INFO  
